CONTRIBUTING.rst

```diff
@@ -68,22 +68,19 @@ Ready to contribute? Here's how to set up `phasespace` for local development.
 
     $ mkvirtualenv phasespace
     $ cd phasespace/
-    $ python setup.py develop
+    $ pip install -e .[dev]
 
 4. Create a branch for local development::
 
     $ git checkout -b name-of-your-bugfix-or-feature
 
    Now you can make your changes locally.
 
-5. When you're done making changes, check that your changes pass flake8 and the
-   tests, including testing other Python versions with tox::
+5. When you're done making changes, check that your changes pass pre-commit and the
+   tests:
 
-    $ flake8 phasespace tests
-    $ python setup.py test or py.test
-    $ tox
-
-   To get flake8 and tox, just pip install them into your virtualenv.
+    $ pytest
+    $ pre-commit run -a
```

Comment:
https://github.com/zfit/phasespace/pull/52#issuecomment-850403410

Oh you mean these changes. Those I did myself, to update the documentation ;)